detox/src/index.test.js

```diff
@@ -17,6 +20,18 @@ describe('index', () => {
     await detox.cleanup();
   });
 
+  it('Basic usage, if init throws, log a clear explanation', async () => {
+    mockDetox.mockImplementation(() => ({
+      init: () => Promise.reject(new Error('Some tragic event occurred'))
+    }));
+    try {
+      await detox.init(schemes.validOneDeviceNoSession);
+    } catch (e) {
+      expect(e).toBeDefined();
```

Comment:
can this ever run without `e` being defined?

Reply:
This checks that the error thrown by detox internally is logged and then
rethrown